_assets/js/laws-regs-parser.js

```diff
@@ -158,3 +155,61 @@ function shareLink(e, divType) {
         shareIcon.style.display = 'block';
     }, 2000);
 }
+
+function addCopyEventListeners(btn, divType) {
+    btn.addEventListener('click', (e) => { copyText(e, divType) });
+    btn.addEventListener('mouseover', (e) => { highlightText(e, divType) });
+    btn.addEventListener('mouseleave', (e) => { unHighlightText(e, divType) });
+}
+
+function addPrintEventListeners(btn, divType) {
+    btn.addEventListener('click', (e) => { printText(e, divType) });
+    btn.addEventListener('mouseover', (e) => { highlightText(e, divType) });
+    btn.addEventListener('mouseleave', (e) => { unHighlightText(e, divType) });
+}
+
+function search() {
+    const searchBox = document.getElementById('searchbox');
+    const searchQuery = searchBox.value.toLowerCase();
+    if (!searchQuery.length){
+        clearSearch();
+        return;
+    };
+    const sections = document.querySelectorAll('.section')
+    for (let i = 0; i < sections.length; i++) {
+        removeHighlights(sections[i]);
+        if(sections[i].innerText.toLowerCase()
```

Comment:
nit: looking at some of the inconsistencies in spacing here:

```
if (!searchQuery.length){
if(sections
...
```

I realize we don't have an `npm run gulp:write` like in portal. WDYT about a tech task for consistent JS formatting?

Reply:
ran a linter in VS code for now!

Reply:
I will add a tech task to the backlog